### DISCUSSION TITLE
pico.css and amcharts

### DISCUSSION THREAD

**Discussion by jobidon** [picocss/pico#626] [Help]
I've noticed that when I combine amcharts 5 with pico.css there is an artifact in the top left corner of every chart produced. This looks like a portion of a button (the chart's zoom button perhaps?) that is hidden away from view in the chart, but because of css rules, a portion of it shows up in the canvas. The artifact persists across browsers, but is not shown when I remove pico from the project.
I tried to unset all css rules for buttons but it does not improve the situation.
I was able to reduce its size by changing the form-element-spacing, but I suspect there might be an easier workaround.
Has anyone encountered this before? How did you resolve it?

**Comment by beisdog** [comment]
lllllllllll
